[2025-03-24T16:35:11.936-0700] {processor.py:186} INFO - Started process (PID=2984) to work on /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:35:11.937-0700] {processor.py:914} INFO - Processing file /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-24T16:35:11.937-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:35:11.937-0700] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:35:11.957-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:35:11.956-0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-24T16:35:11.957-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:35:11.957-0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-24T16:35:11.957-0700] {processor.py:927} WARNING - No viable dags retrieved from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:35:11.997-0700] {processor.py:208} INFO - Processing /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2025-03-24T16:37:23.084-0700] {processor.py:186} INFO - Started process (PID=3458) to work on /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:37:23.086-0700] {processor.py:914} INFO - Processing file /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-24T16:37:23.087-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:37:23.086-0700] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:37:23.105-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:37:23.104-0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-24T16:37:23.105-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:37:23.105-0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-24T16:37:23.105-0700] {processor.py:927} WARNING - No viable dags retrieved from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:37:23.142-0700] {processor.py:208} INFO - Processing /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2025-03-24T16:40:36.595-0700] {processor.py:186} INFO - Started process (PID=4194) to work on /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:40:36.595-0700] {processor.py:914} INFO - Processing file /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-24T16:40:36.596-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:40:36.596-0700] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:40:36.614-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:40:36.614-0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-24T16:40:36.614-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:40:36.614-0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-24T16:40:36.614-0700] {processor.py:927} WARNING - No viable dags retrieved from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:40:36.652-0700] {processor.py:208} INFO - Processing /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2025-03-24T16:43:55.115-0700] {processor.py:186} INFO - Started process (PID=4872) to work on /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:43:55.117-0700] {processor.py:914} INFO - Processing file /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-24T16:43:55.117-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:43:55.117-0700] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:43:55.134-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:43:55.134-0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-24T16:43:55.134-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:43:55.134-0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-24T16:43:55.134-0700] {processor.py:927} WARNING - No viable dags retrieved from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:43:55.171-0700] {processor.py:208} INFO - Processing /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.057 seconds
[2025-03-24T16:47:13.731-0700] {processor.py:186} INFO - Started process (PID=5550) to work on /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:47:13.732-0700] {processor.py:914} INFO - Processing file /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-24T16:47:13.733-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:47:13.732-0700] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:47:13.754-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:47:13.753-0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-24T16:47:13.754-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:47:13.754-0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-24T16:47:13.754-0700] {processor.py:927} WARNING - No viable dags retrieved from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:47:13.797-0700] {processor.py:208} INFO - Processing /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2025-03-24T16:50:34.287-0700] {processor.py:186} INFO - Started process (PID=6263) to work on /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:50:34.288-0700] {processor.py:914} INFO - Processing file /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-24T16:50:34.289-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:50:34.289-0700] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:50:34.307-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:50:34.307-0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-24T16:50:34.308-0700] {logging_mixin.py:190} INFO - [2025-03-24T16:50:34.308-0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-24T16:50:34.308-0700] {processor.py:927} WARNING - No viable dags retrieved from /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-24T16:50:34.345-0700] {processor.py:208} INFO - Processing /Users/jingwenxu/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
